.github/workflows/pa11y.yml

```diff
@@ -0,0 +1,30 @@
+name: pa11yci
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Run pally
+    - run: | 
+      npm install -g pa11y-ci
+      pa11y-ci --config ./awsPa11y.json 2>&1 | tee pa11y_output.txt
+        
+    - name: Read pa11y_output file.
+      id: pa11y_output
+      uses: juliangruber/read-file-action@v1
+      with:
+        path: ./pa11y_output.txt
+
+    - name: Check for pa11y failures.
+      if: contains(steps.pa11y_output.outputs.content, 'Errors in')
+      run: |
+        echo "::error::The site is failing accessibility tests. Please review the comment in the pull request or the pa11y-ci step in the workflow for details."
```

Comment:
Please change the error message as there is no comment left in the PR if the test fails